flyer/src/templates/main.html

```diff
@@ -22,7 +22,7 @@
                 <div class="the-flyer">
 
                     <div class="email-signup" data-component="the-flyer-email-interactive">
-                      <iframe src="https://www.theguardian.com/email/form/plaintone/2211" height="42px" width="100%" scrolling="no" frameborder="0" seamless
+                      <iframe src="https://www.theguardian.com/email/form/plaintone/3806" height="42px" width="100%" scrolling="no" frameborder="0" seamless
```

Comment:
Line 53 will probably need changing too, so that the example matches the victorious format

app.js was responsible for setting the exampleUrl:
https://www.theguardian.com/email/the-flyer?format=email